matchms/similarity/__init__.py

```diff
@@ -1,9 +1,11 @@
 """similarity module"""
 from .CosineGreedy import CosineGreedy
 from .IntersectMz import IntersectMz
+from .ModifiedCosine import ModifiedCosine
 
 
 __all__ = [
     "CosineGreedy",
-    "IntersectMz"
+    "IntersectMz",
+    "ModifiedCosine"
```

Comment:
Add comma so next similarity method only needs to change a single line.
```suggestion
    "ModifiedCosine",
```